registry/remote/referrers.go

```diff
@@ -68,6 +68,11 @@ var (
 	errNoReferrerUpdate = errors.New("no referrer update")
 )
 
+// ErrDanglingReferrerIndex is returned when failed to delete old referrer index
+// after newly updated referrer index being pushed.
+// Only returned if referrer API is unavailable.
+type ErrDanglingReferrerIndex error
```

Comment:
Wondering if it should be
```suggestion
type ErrDanglingReferrersIndex error
```